File: ClientApp/src/models/Album.ts
import {ImgPaths} from './ImgPaths';


export class Album{
    ID: number;
    Title: string;
    ImgPaths : ImgPaths[]=[];
    Description: string;
    UserModelID: number;
  constructor( tittle: string, description: string, userId: number ) {
    this.Title = tittle;
    this.Description = description;
    this.UserModelID=userId;

  }
}

File: Models/Album.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Backend_Foto.Models
{
    public class Album
    {
       
        public int ID { get; set; }
        public string Title { get; set; }
        public List<ImgPath> ImgPaths { get; set; }
        public string Description { get; set; }
        public int UserModelID { get; set; }
      
       
    }
}
